node_cli/realm/realmTasks.js

```diff
@@ -0,0 +1,101 @@
+var Realm = require("realm");
+var mongodb = require("mongodb");
+const schemas = require("./schemas");
+const index = require("../index");
+
+async function setRealm(user) {
```

Comment:
`setRealm` seems wrong. Should this be `getRealm`?

Reply:
`openRealm` ?

Reply:
`openRealm` works.

Side note, I'm not sure if it's performant to re-open the same realm many times as opposed to passing around a singleton. Maybe we could add some caching based on user id if it's an issue?